geps/gep-713.md

```diff
@@ -560,50 +560,536 @@ ties:
+
+Historically, this is a risky strategy and needs to be carefully applied, as
+it's an excellent way to create apiserver load problems, which can produce a large
+range of bad effects for cluster stability.
+
+This does not mean that we can't do anything at all that affects multiple objects,
+but that we need to carefully consider what information is stored in status so 
+that _every_ Policy update does not require a status update.
+
+#### Solution summary
+
+Because Policy Attachment is a pattern for APIs, not an API, and needs to address
+all the problems above, the strategy this GEP proposes is to define a range of
+options for increasing the discoverabilty of Policy resources, and provide
+guidelines for when they should be used.
+
+It's likely that at some stage, the Gateway API CRDs will include some Policy
+resources, and these will be designed with all these discoverabiity solutions
+in mind.
+
+
+### Solution cookbook
+
+This section contains some required patterns for Policy objects and some
+suggestions. Each will be marked as MUST, SHOULD, or MAY, using the standard 
+meanings of those terms.
+
+#### Standard label on CRD objects
+
+Each CRD that defines a Policy object MUST include a label that specifies that
+it is a Policy object.
+
+The label is `gateway.networking.k8s.io/policy: true`.
+
+This solution is intended to allow both users and tooling to identify which CRDs
+in the cluster should be treated as Policy objects, and so is intended to help
+with discoverability generally. It will also be used by the forthcoming `kubectl`
+plugin.
+
+##### Design considerations
+
+This is already part of the API pattern, but is being lifted to more prominience
+here.
+
+#### Standard status Condition
+
+Implementations that use Policy objects MUST put a Condition into `status.Conditions`
+of any objects affected by a Policy.
+
+That Condition must have a `type` of `gateway.networking.k8s.io/PolicyAffected`,
+and have the optional `observedGeneration` field kept up to date when the `spec`
+of the Policy-attached object changes.
+
+For objects that do _not_ have a `status.Conditions` field available (`Secret`
+is a good example), that object MUST instead have an annotation of
+`gateway.networking.k8s.io/PolicyAffected: true` added instead.
+
+
+##### Design Considerations
+The intent here is to add at least a breadcrumb that leads object owners to have
+some way to know that their object is being affected by another object, while
+minimizing the number of updates necessary.
+
+Minimizing the object updates is done by only having an update be necesary when
+the affected object starts or stops being affected by a Policy, rather than if
+the Policy itself has been updated.
+
+There is already a similar Condition to be placed on _Policy_ objects, rather
+than on the _targeted_ objects, so this solution is also being included in the
+Condiions section below.
+
+#### GatewayClass status Extension Types listing
```

Comment:
It might be useful to differentiate between what's already been defined by the GEP and is already fairly solidified (the two solutions above this) with what is still provisional/in progress and will require future GEPs.